apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -27,10 +27,6 @@ API.v1.addRoute(
 	{ authRequired: true, validateParams: isOauthAppsGetParams },
 	{
 		async get() {
-			if (!(await hasPermissionAsync(this.userId, 'manage-oauth-apps'))) {
```

Comment:
I don't think that removing the permission altogether is a good idea
we should probably just have a different one for managing oauth apps and actually fetching them

Reply:
It's been without any permission checks forever before this change was done.
I just reverted that change to get it back to a working state with minimal effort.

I'm proposing to fix it the easy way now and add an additional oauth-fetch-permission later.

Reply:
yea yea I see, but I don't believe this is a good approach overall, this route should be guarded, if it wasn't in the past it probably was a mistake.
but I believe the permission the see an app should not be the same as the permission to manage it though.

I think I can try to push this PR through if we add a new permission, I can help you on that if you need it

Reply:
If you can add that permission for me I'd be happy to add the check back in with the new permission.